libs/libcommon/src/libcommon/parquet_utils.py

```diff
@@ -138,11 +138,30 @@ def is_list_pa_type(parquet_file_path: Path, feature_name: str) -> bool:
     return is_list
 
 
+def truncate_binary_columns(table: pa.Table, max_binary_length: int, features: Features) -> tuple[pa.Table, list[str]]:
```

Comment:
I applied this change in the libviewer PR as well but pilling this out here to make the reviewer process easier.